From a1b59472d207054ff635636807c943c0b14aae58 Mon Sep 17 00:00:00 2001
From: Martin Fick <mfick@codeaurora.org>
Date: Tue, 19 Apr 2011 09:35:08 -0600
Subject: [PATCH] Fix comma/space separation in email templates

Fix the email templates which reversed the ordering for
the comma and space between addressees.

Change-Id: Id12bcc0e91ba8a106016016834fe2d63b299b20d
---
 .../com/google/gerrit/server/mail/NewChange.vm     |    2 +-
 .../google/gerrit/server/mail/ReplacePatchSet.vm   |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gerrit-server/src/main/resources/com/google/gerrit/server/mail/NewChange.vm b/gerrit-server/src/main/resources/com/google/gerrit/server/mail/NewChange.vm
index a9a5fed..ec7028b 100644
--- a/gerrit-server/src/main/resources/com/google/gerrit/server/mail/NewChange.vm
+++ b/gerrit-server/src/main/resources/com/google/gerrit/server/mail/NewChange.vm
@@ -33,7 +33,7 @@
 ## ChangeSubject.vm and ChangeFooter.vm.
 ##
 #if($email.reviewerNames)
-Hello $StringUtils.join($email.reviewerNames, ' ,'),
+Hello $StringUtils.join($email.reviewerNames, ', '),
 
 I'd like you to do a code review.#if($email.changeUrl)  Please visit
 
diff --git a/gerrit-server/src/main/resources/com/google/gerrit/server/mail/ReplacePatchSet.vm b/gerrit-server/src/main/resources/com/google/gerrit/server/mail/ReplacePatchSet.vm
index 356f02e..4575965 100644
--- a/gerrit-server/src/main/resources/com/google/gerrit/server/mail/ReplacePatchSet.vm
+++ b/gerrit-server/src/main/resources/com/google/gerrit/server/mail/ReplacePatchSet.vm
@@ -33,7 +33,7 @@
 ## ChangeEmail: see ChangeSubject.vm and ChangeFooter.vm.
 ##
 #if($email.reviewerNames)
-Hello $StringUtils.join($email.reviewerNames, ' ,'),
+Hello $StringUtils.join($email.reviewerNames, ', '),
 
 I'd like you to reexamine a change.#if($email.changeUrl)  Please visit
 
-- 
1.7.4.1

